### ISSUE TITLE
Multiple files, not only one

### ISSUE THREAD

**Issue by tylerlmn** [rmariuzzo/Lang.js#88]
Hello,

I use Laravel and VueJS and would like to offer my site in different languages. To make it uncomplicated to translate the page, I would like to have different files and not just one.

Currently, it looks like this:
`module.exports = {
    "en.auth": {
        "failed":"These credentials do not match our records :name.",
        "throttle":"Too many login attempts. Please try again in :seconds seconds."
    },
    "de.auth": {
        "failed":"These credentials do not match our records :name.",
        "throttle":"Too many login attempts. Please try again in :seconds seconds."
    },
    "es.auth": {
        "failed":"These credentials do not match our records :name.",
        "throttle":"Too many login attempts. Please try again in :seconds seconds."
    },
}`

I would like to have different files like en.js, de,js and es.js. 

Meine app.js sieht wie folgt aus:
`/* Laravel VueJS Translation */
require('lang.js');
import VueLang from '@eli5/vue-lang-js'
import translations from '../lang/en';
Vue.use(VueLang, {
    messages: translations,
    // locale: 'en', // Set locale
    fallback: 'en' // Set fallback lacale
});`

Do you have any idea how I split a file to three or more? With if, elseif and else it somehow doesn't work. :(

Thanks for your help!

Kind regards,
Tyler

**Comment by rmariuzzo** [comment]
Unfortunately, we don't have this feature out of the box in this package (lang.js) nor Laravel-JS-Localization.

However, I will keep this issue open and see if we can implement that behind a flag.

**Comment by tylerlmn** [comment]
Hey @rmariuzzo ,

thank you very much for your feedback. If I want to have different people working on a translation, do you have any idea how I can handle that with **one file**? I would not to have English translators see the Spanish version.

Should you have no idea, no problem, then I'll wait for add in this package, should it come. 😃 

Kind regards,
Tyler
